### PR TITLE
Added updates and citations for EHR biases (corrected)

### PR DESCRIPTION
**Pull request by laurakwiley**
This is about 70% of what I could include. I'm somewhat inclined to leave this as is so y'all can submit today. The major piece of the literature not included is the work around EHR data quality.

### CODE REVIEW

**Comment by agitter** [comment]
@laurakwiley Thanks!  Don't worry about code climate reports.  Is there a single EHR data quality paper we can cite to act as a placeholder for now?  I don't know that literature.

Please also create an account for the journal submission system and leave it [here](https://github.com/greenelab/deep-review/issues/369#issuecomment-303191214).

**Comment by laurakwiley** [comment]
Yes I can cite one review of the data quality literature. I'll add it and make a pull request.

**Comment by agitter** [comment]
Thanks.  Once you add it, I'll review and merge.

**Comment by laurakwiley** [comment]
Ok I think the update has been added to the pull request and Travid CI build passed.

**Comment by agitter** [comment]
Thanks again.  I'm going to add my review.  Then I'll make a commit on your branch to address some minor things and merge.

File: author_contact_approve.md [hunk 1]
```diff
@@ -25,3 +25,4 @@
 | jisraeli | Johnny Israeli | 0000-0003-1633-5780 | Biophysics Program, Stanford University | jisraeli@stanford.edu | Yes |  |
 | xieconnect | Wei Xie | 0000-0002-1871-6846 | Electrical Engineering and Computer Science, Vanderbilt University | wei.xie@vanderbilt.edu | Yes | |
 | AvantiShri | Avanti Shrikumar | 0000-0002-6443-4671 | Stanford University, Department of Computer Science | avanti@stanford.edu | Yes |  |
+| laurakwiley| Laura K. Wiley | 0000-0001-6681-9754 | Division of Biomedical Informatics and Personalized Medicine, University of Colorado School of Medicine | laura.wiley@ucdenver.edu | Yes | |
```
Comment (agitter) [inline author_contact_approve.md]: I'll manually update our author count.  It isn't part of the automatic build yet.

File: sections/03_categorize.md [hunk 2]
```diff
@@ -425,28 +425,33 @@ costly practice of manual annotation.
...
 biomedical dataset, e.g. clinical trial data.
 
 Even within the same healthcare system, EHRs can be used differently
-[@pmid:24159271 @pmid:21347133]. Individual users have unique usage patterns,
+[@pmid:24159271 @pmid:21347133]. Individual users have unique documentation and ordering patterns,
 with different departments and different hospitals having different priorities
 that code patients and introduce missing data in a non-random fashion
 [@doi:10.1016/S0168-8510(02)00208-7]. Patient data may be kept across several
-"silos" within a single health system. Even the most basic task of matching
+"silos" within a single health system (e.g., separate nursing documentation, registries,
```
Comment (agitter) [inline sections/03_categorize.md]: I dropped commas in #499

File: sections/03_categorize.md [hunk 3]
```diff
@@ -465,7 +470,10 @@ preventing the detection of weak effects
 parameters that can be trained in a model. Further, rules-based algorithms have
 been popular in EHR-based research, but because these are developed at a single
 institution and trained with a specific patient population, they do not transfer
-easily to other populations [@doi:10.1136/amiajnl-2013-001935]. For example,
+easily to other healthcare systems [@doi:10.1136/amiajnl-2013-001935]. Genetic 
+studies using EHR data are subject to even more bias, as the differences in population 
+ancestry across health centers (e.g., proportion of patients with African or Asian
```
Comment (agitter) [inline sections/03_categorize.md]: Comma

File: sections/03_categorize.md [hunk 4]
```diff
@@ -478,7 +486,11 @@ attempts to share and combine EHR data face considerable legal and ethical
 barriers. Patient privacy can severely restrict the sharing and use of EHR
 [@doi:10.1093/ije/dyn022]. Here again, standards are heterogeneous and evolving,
 but often EHR data can often not be exported or even accessed directly for
-research purposes without appropriate consent. Once again, this has the effect
+research purposes without appropriate consent. In the United States, research use 
+of EHR data are subject both to the Common Rule and the Health Insurance Portability 
+and Accountability ACT (HIPPA). Ambiguity in the regulatory language and individual
```
Comment (agitter) [inline sections/03_categorize.md]: Act instead of ACT

**Pull request merged**